backport-Guide.md

```diff
@@ -0,0 +1,97 @@
+# Kata Containers stable backport workflow
+
+* [Backport Workflow](#backport-workflow)
+
+This document provides a short guide on how to perform a stable backport of a
+PR or patchset in the Kata Containers repositories.
+
+It does not cover all eventualities that might be encountered while performing
+a backport, and only documents one potential git based workflow. It does
+serve as a short introduction and reminder of the basic stable backport process.
+
+## Backport Workflow
+
+The basic workflow involves creating a new local branch from the stable tree you
+are targetting, then cherry picking the commits from your master branch PR
+onto that branch. You then submit your branch to GitHub as a PR against the
+stable branch (and not the `master` branch).
+
+The following example shows a pseudo-backport of a PR from the `master` branch
+into the `stable-1.2` branch of the `runtime` repo.
+
+- Ensure your local repo is up to date:
+
+    ```bash
+    $ cd ${GOPATH}/src/github.com/kata-containers/runtime
+    $ git fetch origin
+    ```
+
+    And check the list of current stable branches:
+
+    ```bash
+    $ cd ${GOPATH}/src/github.com/kata-containers/runtime
+    $ git branch -r | grep origin/stable
+    ```
+
+- Create your branch to work on:
+
+    ```bash
+    $ git checkout -b my_1.2_pr_backport origin/stable-1.2
+    ```
+
+- Locate the commits you want to pull in:
+
+    Here we look in the `master` branch, presuming the PR has been merged.
+    If you have the PR in a local branch you can substitute `master` for the name
+    of that branch.
+
+    ```bash
+    $ git log --oneline master
+    # And search for the SHA of the commits you wish to backport.
+    ```
+
+- Pull in your commits:
+
+    If you are pulling the commits in from the `master` branch, you can add the `-x`
+    argument to `git cherry-pick` to automatically add a reference in the
+    commit to the original commits. This is strongly recommended to aid traceability.
+    If you pick the commits from your local branch do *not* use `-x`. This
+    potentially adds references to SHAs that only exist in your user branches, which
+    is not be useful for future tracability.
+
+    Also, it is required you add the `-s` signoff to the commits, if you did not
+    create the original commits.
+
+    ```bash
+    $ git cherry-pick -x <commit>...
+    ```
+
+    You can cherry pick ranges of commits, etc. Please see the `git-cherry-pick(1)`
+    man page for more information.
```

Comment:
It might be worth stating here that since the commit(s) already existing in master, it is *not* necessary to create a new issue since the "Fixes #XXX" comment can be the same - aka all PRs across all branches that fix a particular problem will "hang off" / be associated with that single issue (https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#main-patch).